python/core/qgsrange.sip

```diff
@@ -147,15 +147,19 @@ class QgsTemporalRange
 %End
   public:
 
-    QgsTemporalRange( const T &begin, const T &end, bool includeBeginning = true, bool includeEnd = true );
+#if SIP_VERSION >= 0x41303
 %Docstring
  Constructor for QgsTemporalRange. The ``begin`` and ``end`` are specified,
  and optionally whether or not these bounds are included in the range.
```

Comment:
Can you rephrase that to something like

``begin`` and ``end`` can optionally be passed to specify if bounds are included in the range.